src/current/releases/index.md

```diff
@@ -264,18 +265,27 @@ The following binaries are not suitable for production environments:
             </td>
             <td>{{ r.release_date }}</td> {% comment %} Release date of the release. {% endcomment %}
             <td>
-                {% if r.withdrawn == true %} {% comment %} Suppress withdrawn releases. {% endcomment %}
+            {% if r.withdrawn == true %} {% comment %} Suppress withdrawn releases. {% endcomment %}
                 <span class="badge badge-gray">Withdrawn</span>
+            {% else %}
+                {% if r.source == true %}
+                <b>{% if r.docker.docker_arm == false %}Intel{% else %}Multi-platform{% endif %}</b>: <code>{{ r.docker.docker_image }}:{{ r.release_name }}</code>
                 {% else %}
-                    {% if r.source == true %}
-                <b>Intel{% if r.docker.docker_arm == true %}/ARM{% endif %}</b>: <code>{{ r.docker.docker_image }}:{{ r.release_name }}</code>{% if r.docker.docker_arm_experimental == true %} (Experimental){% endif %}{% if r.docker.docker_arm_limited_access == true %} (Limited Access){% endif %}
-                    {% else %}
                 N/A
-                    {% endif %}
+                {% endif %}
+            {% endif %}
+            </td>
+            <td>
+            {% if r.docker.docker_arm_limited_access == true %}
+              **Intel**: Generally available<br />**ARM**: Limited Access
+            {% elsif r.docker.docker_arm_experimental == true %}
+              **Intel**: Generally available<br />**ARM**: Experimental
+            {% else %}
+              Generally available
+            {% endif %}
```

Comment:
WDYT of updating the first mention of "generally available" *on line 17* to add " (GA)", and then changing these later mentions to "GA"?

I think this would streamline the content of the table, make it look better at narrower widths, and may also make it easier for the user to copy the docker tag.